packages/sdk/src/dialect-cloud-api/data-service-wallets-api.v1.ts

```diff
@@ -0,0 +1,33 @@
+import type { Token } from '../auth/auth.interface';
+import {
+  createHeaders,
+  withReThrowingDataServiceError,
+} from './data-service-api';
+import type { WalletDto } from './data-service-dapps-api';
+import axios from 'axios';
+
+export interface DataServiceWalletsApiV1 {
+  upsertWallet(walletDto: WalletDto, token: Token): Promise<WalletDto>;
+}
+
+export class DataServiceWalletsApiClientV1
+  implements DataServiceWalletsApiV1
+{
+  constructor(
+    private readonly baseUrl: string,
+  ) {}
+
+  async upsertWallet(walletDto: WalletDto, token: Token): Promise<WalletDto> {
```

Comment:
I think this dto is not needed at all, since it's not defined in data-service controller